template.yaml

```diff
@@ -27,7 +27,7 @@ Metadata:
         "amplify",
       ]
     HomePageUrl: https://github.com/aws-samples/cloudfront-authorization-at-edge
-    SemanticVersion: 1.0.8
+    SemanticVersion: 1.0.9
```

Comment:
Should be 1.0.10

Stuff moves quickly :D

Reply:
Updated